e2e/config/microlcConfig.json

```diff
@@ -6,7 +6,8 @@
     },
     "logo": {
       "alt": "Mia Care",
-      "url_light": "https://media-exp1.licdn.com/dms/image/C4D0BAQEf8hJ29mN6Gg/company-logo_200_200/0/1615282397253?e=2159024400&v=beta&t=tQixwAMJ5po8IkukxMyFfeCs-t-zZjyPgDfdy12opvI"
+      "url_light_image": "https://media-exp1.licdn.com/dms/image/C4D0BAQEf8hJ29mN6Gg/company-logo_200_200/0/1615282397253?e=2159024400&v=beta&t=tQixwAMJ5po8IkukxMyFfeCs-t-zZjyPgDfdy12opvI",
```

Comment:
I see this is a configuration for Mia Care and for this reason I imagine this link (and others in the PR as I can see) refers to the logo of the company (mia-care).

Since this is a mia-platforrm project can we use something non related to another brand? And define the mock-server with fake company names? 

If not in this PR I'd open an issue and clean the mock-server from any reference to other existing companies

Reply:
I'll take care of it in this PR if it's not a problem using as fake company name 'My Company'  and images not related to other existing brands except mia-platform. Is it ok?